### PR TITLE
Game of Links rules

### PR DESCRIPTION
**Pull request by SaveTheAles**
Things todo: 
- [ ] Intro
- [ ] Conclusion
- [ ] Values and digits
- [ ] Auction explanation
- [ ] Rank calculation explanation
- [ ] Rank discipline overview (it's very raw)
- [ ] Probably add testnet genesis file pie chart

### CODE REVIEW

File: drafts/GoL-reules/Game of Links.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+# The Game of Links. Goals, rules and provision.
+
+## Intro
+
+
+## General provision of the Game.
+
+![](euler-5_timeline.png)
```
Comment (xhipster) [inline drafts/GoL-reules/Game of Links.md]: The structure is far from ideal. The need clear sequence of events including testing GoT and cyber•Auction tests...

Comment (xhipster) [reply drafts/GoL-reules/Game of Links.md]: 600k ATOMs is a goal now

File: drafts/GoL-reules/Game of Links.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+We want to reproduce the whole process of mainnet launch in `Euler-5` for testing auction, drop and distribution models. As shown in figure above the Game isn't start with `Euler-5` launch. We want to allow some time to participants of the Game for their funds checking and hardware and software setuping. Also, in that 15-days limb the Gamers can decide on what exactly discipline they want to participate and strategize.
+
+`Euler-5` Genesis file will contain all drops to Ethereum and Cosmos communities. Furthermore,  it will contain validators lifetime rewards from `Euler-4`. For new incomers, we'll propose to get CYB tokens at the Auction just for less. In the Auction, newbies can buy CYBs or GOLs for uAtoms and Szabos. 
+
+> ***Few words about the Auction.***
+
+With the Game of Links start after 15 days of `Euler-5` launch, we will start tracking the progress at the Game monitor and store data. Then, and only then. The monitor will be available at the Brain block explorer by cyberd.ai address.
```
Comment (xhipster) [inline drafts/GoL-reules/Game of Links.md]: I think that cyberd.ai changes must be optional. I dont sure that we will be able to deliver this ahead. Probably during...

File: drafts/GoL-reules/Game of Links.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+We want to reproduce the whole process of mainnet launch in `Euler-5` for testing auction, drop and distribution models. As shown in figure above the Game isn't start with `Euler-5` launch. We want to allow some time to participants of the Game for their funds checking and hardware and software setuping. Also, in that 15-days limb the Gamers can decide on what exactly discipline they want to participate and strategize.
+
+`Euler-5` Genesis file will contain all drops to Ethereum and Cosmos communities. Furthermore,  it will contain validators lifetime rewards from `Euler-4`. For new incomers, we'll propose to get CYB tokens at the Auction just for less. In the Auction, newbies can buy CYBs or GOLs for uAtoms and Szabos. 
+
+> ***Few words about the Auction.***
+
+With the Game of Links start after 15 days of `Euler-5` launch, we will start tracking the progress at the Game monitor and store data. Then, and only then. The monitor will be available at the Brain block explorer by cyberd.ai address. 
+
+The end of the Game will define one of 3 control events happens as a result:
```
Comment (xhipster) [inline drafts/GoL-reules/Game of Links.md]: Lets make it 2of3

File: drafts/GoL-reules/Game of Links.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+-  90 days will be passed after the Game start
+-  500.000 ATOMs well be donated after funding start
```
Comment (xhipster) [inline drafts/GoL-reules/Game of Links.md]: This

File: drafts/GoL-reules/Game of Links.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+-  Knowledge Graph well be increased by 1B links
+
+In addition to each requirement we add one more: set of `146` active validators must have lasted at least 10.000 blocks. In another case, the Game would be considered as unsuccessful and restarted.
```
Comment (xhipster) [inline drafts/GoL-reules/Game of Links.md]: We discissed with @litvintech that we could set a hard requirement to 73 and assign bonus for reaching 146

File: drafts/GoL-reules/Game of Links.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+There are no limits in disciplines quantity participation. By all means. 
+You can set up your own validator to get a reward for validator's uptime or you can link CIDs  and load network bandwidth. You can create a promotion around your validator and collect stakes from users. You can do all of those things together. Just play and monitor the results. 
+
+**3. Monitor funding process.**
+
+The mainnet Genesis file depends on Funding process in `Euler-5`, in particular, how many tokens will allocate for the Game of Links results. Here is an example table of possible results of funding process:
+
+ATOMs donated | % of desireble | LT reward | Load reward | Stake reward | Link reward | 
+--- | --- | --- | --- |--- |--- |
+500,000 | 100 | 3 TCYB | 3 TCYB | 3 TCYB | 6 TCYB |
```
Comment (xhipster) [inline drafts/GoL-reules/Game of Links.md]: Lack of 600k case

File: drafts/GoL-reules/Game of Links.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+400,000 | 80 | 2.4 TCYB | 2.4 TCYB | 2.4 TCYB | 4.8 TCYB |
+200,000 | 40 | 1.2 TCYB | 1.2 TCYB | 1.2 TCYB | 2.4 TCYB |
+100,000 | 20 | 0.6 TCYB | 0.6 TCYB | 0.6 TCYB | 1.2 TCYB |
+
+Other possible cases calculated the same way. 
+
+Participants, which want to get CYBs in Genesis file of mainnet but cannot play, can donate ATOMs, therefore, increasing the percentage of payouts for other participants and for themselves. The relationship between donated ATOMs and percentage of CYBs allocated you can see below:
+
+![funding](funding.jpg)
```
Comment (xhipster) [inline drafts/GoL-reules/Game of Links.md]: I think we must illustrate share of allocation for rewards as the 3d line

File: drafts/GoL-reules/Game of Links.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
+# The Game of Links. Goals, rules and provision.
+
+## Intro
```
Comment (xhipster) [inline drafts/GoL-reules/Game of Links.md]: I think we should clearly translate that the purpose of the game: to prepare and test infrastructure for main network

**Comment by xhipster** [comment]
@SaveTheAles Any update?

**Comment by SaveTheAles** [comment]
@xhipster 
I've been stuck with some researches about euler-5 modeling. Have a discussion with @litvintech
I'll add my commits tomorrow

File: blog/releases/gol-rules/game-of-links.md [hunk 9]
```diff
@@ -0,0 +1,195 @@
...
+## Takeoff funding and disciplines incentives
+
+Takeoff funding has 3 key goals:
+- bootstraping genesis validator set
+- definition of allocation for seed investment participants
+- definition of monetary allocation to Game of Links participants
+
+The development of the project has been done thanks to investments of 8 people. ~3200 ETH has been spend during almost 3 years of development. For that at least 4% CYB is allocated for them by means of THC from cyber•Congress allocation. In order to protect seed investors their share will be defined based on the amount raised during takeoff round. In essence the curve defines allocation of up to 6% CYB for takeoff participants. If takeoff round get 600k ATOMs than seed investors get 4% of CYB by the means of THC and takeoff participants get 6% of CYB directly. If zero amount will be raised seed investors get 10%. The interesting thing is that if amount raised will be below 100k ATOMs participants will have better price when seed investors. In such case seed investors get compensation by gaining more stake though.
+
+![funding](funding.jpg)
```
Comment (xhipster) [inline blog/releases/gol-rules/game-of-links.md]: @SaveTheAles Formula needed

**Pull request merged**